RepresentsMe/ViewControllers/Events/EventDetails/EventCreateViewController.swift

```diff
@@ -168,8 +220,8 @@ class EventCreateViewController: UIViewController,
     /// - Parameter address:    the Address for the Event
     private func set(location:CLLocationCoordinate2D, address:Address) {
         selectedLocation = location
-        selectLocationButton.setTitle(address.addressLine1(),
-                                      for: .normal)
+        selectedLocationLabel.text = "\(address.addressLine1())\n\(address.city), \(address.state)"
```

Comment:
Doesn't have to be done now but we should probably move the ways to format an address to `Address` itself.